core/src/main/scala/org/typelevel/sbt/TypelevelPlugin.scala

```diff
@@ -76,24 +76,32 @@ object TypelevelPlugin extends AutoPlugin {
         java <- githubWorkflowJavaVersions.value.tail // default java is head
       } yield MatrixExclude(Map("scala" -> scala, "java" -> java.render))
     },
-    GlobalScope / tlCommandAliases ++= {
+    GlobalScope / tlPrePRSteps ++= {
       val header = tlCiHeaderCheck.value
       val scalafmt = tlCiScalafmtCheck.value
       val javafmt = tlCiJavafmtCheck.value
       val scalafix = tlCiScalafixCheck.value
+      val mima = tlCiMimaBinaryIssueCheck.value
 
-      val prePR = List("project /", "githubWorkflowGenerate") ++
+      List("githubWorkflowGenerate") ++
         List("+headerCreateAll").filter(_ => header) ++
         List("+scalafixAll").filter(_ => scalafix) ++
         List("+scalafmtAll", "scalafmtSbt").filter(_ => scalafmt) ++
-        List("javafmtAll").filter(_ => javafmt)
+        List("javafmtAll").filter(_ => javafmt) ++
+        List("compile", "test") ++
+        List("mimaReportBinaryIssues").filter(_ => mima)
```

Comment:
See some discussion here.
- https://github.com/typelevel/sbt-typelevel/issues/204

Reply:
Gotcha, this wasn't clear in the existing docs on prePR but I'm happy to remove mima to go back into line with original intent.  Should I remove `compile` and `test` as well?  The docs seemed to indicate that at least `compile` was run?  https://typelevel.org/sbt-typelevel/faq.html#how-do-i-locally-prepare-my-pr-for-ci

> sbt-typelevel comes with a prePR command, which updates the GitHub workflow, generates headers, runs scalafmt, and clean compiles your code.

Reply:
Oops, looks like the docs are not up to date. The original implementation did do a clean compile but we decided to change that in the linked issue.

Reply:
Reverted it back to the original, removing mima and compile.